cmd/brew-test-bot.rb

```diff
@@ -574,6 +574,13 @@ def download
       @formulae += @added_formulae + @modified_formulae
       @test_brew = (!@tap || @test_bot_tap) &&
                    (@formulae.empty? || @test_default_formula)
+
+      puts <<~EOS
+
+        Bottles to be built for:
```

Comment:
Bottles aren't necessarily going to be built. How about `Formula changes to be tested:`

Reply:
Sounds good to me, I'll change that.